Documentation/bpf/toolchain.rst

```diff
@@ -629,7 +629,7 @@ A full command line example with llc's ``-mcpu=probe``:
     $ llc xdp-example.bc -march=bpf -mcpu=probe -filetype=obj -o xdp-example.o
 
 Generally, LLVM IR generation is architecture independent. There are
-however a few differences when using ``clang --target=bpf`` versus
+however, a few differences when using ``clang --target=bpf`` versus
```

Comment:
“There are however,” sounds weird to me if you only have a comma _after_ “however”. Wouldn't you need one before as well? “There are, however,” or even “However/Nonetheless, there are a few differences...”.